web/v1/redshift/sql-runner/sql/standard/02-page-views/01-main/04-engaged-time.sql

```diff
@@ -37,7 +37,7 @@ AS(
 	FROM {{.scratch_schema}}.events_staged{{.entropy}} AS ev
 
 	WHERE ev.event_name = 'page_ping'
-    AND ev.collector_tstamp >= (SELECT lower_limit FROM {{.scratch_schema}}.pv_run_limits{{.entropy}})
+    AND ev.collector_tstamp >= (SELECT lower_limit FROM {{.scratch_schema}}.pv_run_limits{{.entropy}}) -- Limits here aren't necessary?
```

Comment:
Haven't we already filtered `ev` by this stage? I think I agree with this comment, as always though removing this seems scary.

Reply:
Actually, I never meant to commit this comment here, I will remove as BQ release shouldn't touch Redshift source code. 

Rather a fix is scheduled for follow-up Redshift release: https://github.com/snowplow/data-models/issues/5

Ps. Yeah we are querying a filtered table here, it's safe to remove the limit

Reply:
I was so deep into the PR when I stumbled across this I didn't even notice it was the bottom of a RedShift file 🤦🏻

Reply:
I'm glad you caught it!

Reply:
Note to self: this needs a rebase to remove the change to this file - so TODO after all other comments addressed